config/deploy.ignore

```diff
@@ -1,23 +1,36 @@
+.circleci/
+.DS_store
+.editorconfig
+.eslintrc.json
 .git/
-selenium*
-fabfile.py*
+.github/
+.gitignore
+.prettierrc
+.stylelintrc
+.vscode
 *.pyc
+CHANGELOG.md
+CODE_OF_CONDUCT.md
+config/secrets.py.example
+CONTRIBUTING.md
 data/
 docs/
 env/
+fabfile.py*
 git_modules/
 grunt-tasks/
-node_modules/
 Gruntfile.js
+icon-licenses/
 Makefile
+node_modules/
+package-lock.json
 package.json
 README.md
-session.db
+run.py
```

Comment:
What's the rationale for removing this? I've used it in the past to quickly see if I could get the app to run on staging, when debugging weird issues.

(I guess we can still start the app via `flask run` with `FLASK_APP` or similar?)

Reply:
Ah! I didn't you were using it locally. The server is not using it indeed in a normal operation. 

You could run indeed with

```
FLASK_APP=webcompat flask run
```

I wonder if we should get rid of run.py locally too to avoid confusion and be closer from what the server needs.

Reply:
ah wait…
https://github.com/webcompat/webcompat.com/blob/c58c8ee45cc327bd585114c86b103ae6dbaf318e/.circleci/config.yml#L59

Reply:
and 

https://github.com/webcompat/webcompat.com/blob/c58c8ee45cc327bd585114c86b103ae6dbaf318e/package.json#L75-L76

and 

https://github.com/webcompat/webcompat.com/blob/c58c8ee45cc327bd585114c86b103ae6dbaf318e/tests/functional/lib/setup.js#L25-L39

Reply:
Let me remove this from the deploy.ignore.